utils/team_utils.jsx

```diff
@@ -3,22 +3,42 @@
 
 // Use when sorting multiple teams by their `display_name` field
 function compareTeamsByDisplayName(locale, a, b) {
-    if (a.display_name !== b.display_name) {
-        return a.display_name.localeCompare(b.display_name, locale, {numeric: true});
+    if (a.display_name !== null) {
+        if (a.display_name !== b.display_name) {
+            return a.display_name.localeCompare(b.display_name, locale, {numeric: true});
+        }
     }
 
     return a.name.localeCompare(b.name, locale, {numeric: true});
 }
 
 // Use to filter out teams that are deleted and without display_name, then sort by their `display_name` field
-export function filterAndSortTeamsByDisplayName(teams, locale) {
+export function filterAndSortTeamsByDisplayName(teams, locale, teamsOrder = '') {
```

Comment:
Could you add some tests for this function? Those would help verify its behaviour in the future.

Reply:
@nevyangelova did you address this comment? I didn't see anything in the commit. Maybe we file it as an improvement?

Reply:
I did add a test 🤔 More like I updated the existing one.